chapters/tutorials/index.rst

```diff
@@ -34,45 +34,48 @@ And we need the following dependencies:
     pip install torch==1.7.1
 
 
-Once you have Jina and the dependencies installed, let's take a broad overview of what we should do:
+Once you have Jina and the dependencies installed, let's get a broad overview of the process we'll follow:
 
 .. image:: res/flow.png
    :width: 600
 
-At the beginning of the Flow, you have your data, and this can be any type:
+If you see the image, at the beginning of this flow process, you have your data, and this can be any type:
```

Comment:
You see from this image that you have your data at the beginning of this Flow process, and this can be any data type